### ISSUE TITLE
Symlinking priv/static on deploy

### ISSUE THREAD

**Issue by GBH** [edeliver/edeliver#215]
I tried what's outlined in https://github.com/boldpoker/edeliver/wiki/Symlink-static-for-nginx-or-apache with no success. `post_erlang_update` is not even something that's listed in https://github.com/boldpoker/edeliver/wiki/Run-additional-build-tasks

Is that entry out of date? 

What hook do I need that triggers when I do `mix edeliver deploy release`?

Thanks

**Comment by DavidOliver** [comment]
`post_erlang_update` is only called in [erlang_update](https://github.com/boldpoker/edeliver/blob/990ee09e75140697c4ec9e97662b6f0d9f74ef88/strategies/erlang-update#L125), and I'm thinking this means that [the static symlinking example in the wiki](https://github.com/boldpoker/edeliver/wiki/Symlink-static-for-nginx-or-apache) doesn't trigger with deployments of full releases (as opposed to updates).

This is [erlang_deploy_release](https://github.com/boldpoker/edeliver/blob/990ee09e75140697c4ec9e97662b6f0d9f74ef88/strategies/erlang-deploy-release), and it doesn't seem to have any function hooks to use. Could it have `__exec_if_defined "pre_erlang_deploy_release"` and `__exec_if_defined "post_erlang_deploy_release"` added? I believe this would be something I could use, too, as I don't currently use updates, only full releases.

I suppose [the build tasks wiki page](https://github.com/boldpoker/edeliver/wiki/Run-additional-build-tasks) doesn't list `pre|post_erlang_update` because the update process is not part of the build phase.

**Comment by DavidOliver** [comment]
@boldpoker, would it make sense to have the `pre` and `post` `_erlang_deploy_release` hooks?

**Comment by tmepple** [comment]
I'm using a fork based on [this PR](https://github.com/boldpoker/edeliver/pull/200) which adds a simple hook right after `remote_extract_release_archive()`

Then in my `.deliver/config` file, defining this function works great:
```
post_extract_release_archive() {
    status "Creating Static Assets Symlink for Release"
    __sync_remote "
        ln -sfn $DELIVER_TO/$APP/lib/$APP-$VERSION/priv/static $DELIVER_TO/$APP/static
    "
}
```

**Comment by GBH** [comment]
After #200 got merged it's working!
